tools/env-loader/pkg/loaders/yaml.go

```diff
@@ -0,0 +1,93 @@
+/*
+ *  Copyright 2023 Gravitational, Inc
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package loaders
+
+import (
+	"github.com/getsops/sops/v3/decrypt"
+	sopsYaml "github.com/getsops/sops/v3/stores/yaml"
```

Comment:
```suggestion
	sopsyaml "github.com/getsops/sops/v3/stores/yaml"
```

General idiom is to keep package names (even when aliased) all lower case.